changes/2023-09-26_encryption-context-decrypt/change.md

```diff
@@ -0,0 +1,57 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Check Required Encryption Context Keys exist in Encryption Context in Decrypt and the values match in the stored Encryption Context in the message header
+
+## Affected Implementations
+
+This serves as a reference for all implementations that this change affects.
+
+| Language   | Repository                                                                                                     |
+| ---------- | -------------------------------------------------------------------------------------------------------------- |
+| Java       | [aws-encryption-sdk-java](https://github.com/aws/aws-encryption-sdk-java)                                      |
+| .NET       | [aws-encryption-sdk-net](https://github.com/aws/aws-encryption-sdk-dafny/tree/mainline/aws-encryption-sdk-net) |
+| Python     | [aws-encryption-sdk-python](https://github.com/aws/aws-encryption-sdk-python)                                  |
+| CLI        | [aws-encryption-sdk-cli](https://github.com/aws/aws-encryption-sdk-cli)                                        |
+| C          | [aws-encryption-sdk-c](https://github.com/aws/aws-encryption-sdk-c)                                            |
+| Javascript | [aws-encryption-sdk-javascript](https://github.com/aws/aws-encryption-sdk-javascript)                          |
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Summary
+
+The [Decrypt](../../client-apis/decrypt.md) API does not specify
+how it should deal with required encryption context keys it receives from the call to the
+Cryptographic Materials Manager, the encryption context it optionally takes in, and the stored
+encryption context on the message header.
+
+Decrypt MUST verify that if the keys in the required encryption context keys exist in the reproduced encryption context
+AND they are stored in the message header, they MUST have the same value as the reproduced encryption context supplied.
```

Comment:
This is not how you have change the spec:

```
- If a [required encryption context key](../framework/structures.md#required-encryption-context-keys) in
  the decryption materials' response exists in the [message header body](../data-format/message-header.md#header-body)
  the value in the decryption materials' encryption context MUST equal the value stored in the message header body.
```

If you want what you have specified, then I think you need:

```suggestion
For required encryption context keys the value in stored in the message header
MUST equal the value in decryption materials encryption context.
```